src/vtools-analysis.py

```diff
@@ -113,7 +114,7 @@ def summarize(infile, df, config_dict, debug):
         vals.append(val)
     # add derived values
     if "pkt_duration_time" in df:
-        key = "file_duration_time"
+        key = "video_duration"
```

Comment:
video_duration_time for coherence with the input name

Reply:
sure, will update